--- a/bash
+++ b/bash
@@ -117,2 +117,3 @@
-         info list needed nodeps assume-installed print refresh recursive search sysupgrade'
-        'c g i l p s u w y')
+         info list needed nodeps assume-installed print refresh recursive search sysupgrade
+         build keyserver noaur source'
+        'c g i l n p s u w y z')
@@ -131,1 +132,1 @@
-    ! $prev = --@(cachedir|color|config|dbpath|help|hookdir|gpgdir|logfile|root|version) ]]
+    ! $prev = --@(cachedir|color|config|dbpath|help|hookdir|gpgdir|keyserver|logfile|root|version) ]]
@@ -151,1 +152,1 @@
-          _pacman_pkg Slq;;
+          _arch_compgen "`pakku -Ssq "$cur" 2> /dev/null`";;
@@ -164,3 +165,1 @@
-complete -F _pacman pacman
-complete -F _makepkg makepkg
-complete -F _pacman_key -o default pacman-key
+complete -F _pakku -o default pakku
